attributes/default.rb

```diff
@@ -97,13 +97,24 @@
 default['openldap']['slapd_provider'] = nil
 default['openldap']['slapd_replpw'] = nil
 default['openldap']['slapd_rid'] = 102
-default['openldap']['syncrepl_interval'] = '01:00:00:00'
-default['openldap']['syncrepl_type'] = 'refreshAndPersist'
-default['openldap']['syncrepl_filter'] = '(objectClass=*)'
-default['openldap']['syncrepl_use_tls'] = 'no' # yes or no
```

Comment:
Hi Lamont, I took those setting from the existing slapd.conf.erb file and I made sure my changes did not effect the resulting slapd.conf file. So they should already be backwards compatible.